apps/api/src/app/invites/usecases/invite-member/invite-member.usecase.ts

```diff
@@ -33,13 +33,15 @@ export class InviteMember {
     const token = createGuid();
 
     if (process.env.NOVU_API_KEY && (process.env.NODE_ENV === 'dev' || process.env.NODE_ENV === 'production')) {
-      const novu = new Novu(process.env.NOVU_API_KEY);
-      // cspell:disable-next-line
-      await novu.trigger(process.env.NOVU_TEMPLATEID_INVITE_TO_ORGANISATION || 'invite-to-organization-wBnO8NpDn', {
-        to: {
-          subscriberId: command.email,
-          email: command.email,
-        },
+      const novu = new Novu({ apiKey: process.env.NOVU_API_KEY });
+      await novu.trigger({
+        name: process.env.NOVU_TEMPLATEID_INVITE_TO_ORGANISATION || 'invite-to-organization-wBnO8NpDn',
+        to: [
```

Comment:
@tatarco has to become an array in the novu.trigger function? I just noticed.

Reply:
that's the openApi spec definition and our DTO as the server expect it, the fact that it worked before is nothing shy of a miracle